applications/openshift/kubelet/kubelet_configure_tls_cert/rule.yml

```diff
@@ -30,3 +33,4 @@ ocil: |-
     Verify that a certificate path is configured.
     <pre>$ sudo ls /etc/origin/node/certificates/kubelet-client-current.pem</pre>
     Verify that a client certificate is configured.
+
```

Comment:
@shawndwells please remove this empty line that was added.

Reply:
done